apps/system/lockscreen/js/lockscreen_inputpad.js

```diff
@@ -111,25 +116,111 @@
       case 'lockscreen-inputappclosed':
         this.updatePassCodeUI();
         break;
+      case 'touchstart':
+        // TODO: ignore while on error timeout
+        touch = evt.changedTouches[0];
+        target = document.elementFromPoint(touch.clientX, touch.clientY);
```

Comment:
I started out with .target, but unlike changedTouches it does not stay in sync with where the finger is touching on touch moves. IIRC, .target remains where the touch first went down. Hence I went for the alternative that always works.